pki/pki.go

```diff
@@ -944,6 +949,11 @@ func (p *PKI) Save(opt ...ConfigOption) error {
 		if err = fileutil.WriteFile(p.defaults, b, 0644); err != nil {
 			return errs.FileError(err, p.defaults)
 		}
+		if p.profileDefaults != "" {
+			if err = fileutil.WriteFile(p.profileDefaults, []byte("{}"), 0644); err != nil {
+				return errs.FileError(err, p.profileDefaults)
+			}
+		}
```

Comment:
This will ask if we want to overwrite that file, if we don't it will cancel the rest.

I think is going to be very common to use the same profile on multiple CAs. This scenario will probably affect us more than our users. So if the file exists we should do nothing.

Reply:
And do we need this file in the helm template? You don't have any changes in helm.go.